.mailmap

```diff
@@ -0,0 +1,110 @@
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@05709c45-44f0-0310-885b-81a1db45b4a6>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@users.noreply.github.com>
+Jeff Mellen <jeffm@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Jesse Corrington <jesse@glencoesoftware.com> <jesse.corrington@gmail.com>
+Josh Moore <josh@openmicroscopy.org> <j.a.moore@dundee.ac.uk>
+Josh Moore <josh@openmicroscopy.org> <josh@glencoesoftware.com>
+Josh Moore <josh@openmicroscopy.org> <josh@example.com>
+Josh Moore <josh@openmicroscopy.org> <josh.moore@gmx.de>
+Josh Moore <josh@openmicroscopy.org> <josh@05709c45-44f0-0310-885b-81a1db45b4a6>
+Josh Moore <josh@openmicroscopy.org> <jmoore@05709c45-44f0-0310-885b-81a1db45b4a6>
+Josh Moore <josh@openmicroscopy.org> <jmoore@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Josh Moore <josh@openmicroscopy.org> <jmoore@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Joshua Ballanco <jballanc@glencoesoftware.com> Josh Ballanco <jballanc@glencoesoftware.com>
+Joshua Ballanco <jballanc@glencoesoftware.com> <jballanc@gmail.com>
+Kenny Gillen <k.h.gillen@dundee.ac.uk> Kenneth Gillen <k.h.gillen@dundee.ac.uk>
+Kouichi Nakamura <kouichi.c.nakamura@gmail.com> Kouichi C. Nakamura <kouichi.c.nakamura@gmail.com>
+Kouichi Nakamura <kouichi.c.nakamura@gmail.com> KOUICHI NAKAMURA <kouichi.c.nakamura@gmail.com>
+Liza Unson <lunson@glencoesoftware.com> Liza <lunson@glencoesoftware.com>
+Luca Lianas <luca.lianas@crs4.it> <lianas@crs4.it>
+Mark Carroll <m.t.b.carroll@dundee.ac.uk> <M.T.B.Carroll@Dundee.ac.uk>
+Mark Carroll <m.t.b.carroll@dundee.ac.uk> <mtbc@ls28101>
+Muhanad Zahra <muhanad@glencoesoftware.com> <Muhanad.Zahra@outlook.com>
+Melissa Linkert <melissa@glencoesoftware.com> <melissa.linkert@gmail.com>
+Melissa Linkert <melissa@glencoesoftware.com> <mlinkert-x@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Melissa Linkert <melissa@glencoesoftware.com> <melissa@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Melissa Linkert <melissa@glencoesoftware.com> <melissa@05709c45-44f0-0310-885b-81a1db45b4a6>
+Nikolaus Ehrenfeuchter <nikolaus.ehrenfeuchter@unibas.ch>
+Nikolaus Ehrenfeuchter <nikolaus.ehrenfeuchter@unibas.ch> <mail@he1ix.org>
+Paula Forbes <pforbes@computing.dundee.ac.uk> <uimage@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Paul van Schayck <polleke@gmail.com> pvc-local <polleke@gmail.com>
+Petr Walczysko <p.walczysko@dundee.ac.uk> pwalczysko <p.walczysko@dundee.ac.uk>
+Riad Gozim <r.gozim@dundee.ac.uk> <rgozim@users.noreply.github.com>
+Riad Gozim <r.gozim@dundee.ac.uk> <rgozim1@gmail.com>
+Roger Leigh <r.leigh@dundee.ac.uk> <rleigh@debian.org>
+Roger Leigh <r.leigh@dundee.ac.uk> <rleigh@dundee.ac.uk>
+Roger Leigh <r.leigh@dundee.ac.uk> <rleigh@codelibre.net>
+Scott Littlewood <sylittlewood@dundee.ac.uk> <scottlittlewood@gmail.com>
+Sébastien Besson <sbesson@glencoesoftware.com> <seb.besson@gmail.com>
```

Comment:
nitpicking 
```
Sébastien Besson <sbesson@glencoesoftware.com> Sebastien Besson <seb.besson@gmail.com>
```
We have both, with and without accent

Reply:
Correct, so this entry maps all commits authored using `seb.besson@gmail.com` as the email address to use `Sébastien Besson <sbesson@glencoesoftware.com>` as the canonical real name and email address.

Similar to the conversation above, the following 2 lines will achieve the same mapping:

```
Sébastien Besson <sbesson@glencoesoftware.com> Sebastien Besson <seb.besson@gmail.com>
Sébastien Besson <sbesson@glencoesoftware.com> Sébastien Besson <seb.besson@gmail.com>
```

But I am still unconvinced of the value of using this verbose strategy

Reply:
I won't worry about that one. 
The rest looks clearer now in my opinion 
Thanks